res/css/views/rooms/_EventTile.pcss

```diff
@@ -1316,18 +1342,6 @@ $left-gutter: 64px;
                     inset-block-start: -2rem;
                 }
             }
-
-            .mx_EventTile_content .markdown-body {
-                p,
-                ul,
-                ol,
```

Comment:
I am not sure if the margin-bottom for `ul` and `ol` on compact modern layout has been expected, as the other layouts do not have margin-bottom.